### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by Arux-BTT**
- fix an unexpected behavior: when Aeon is selected, if you reload the page, generated address switch to a monero address while aeon is still selected (could be confusing for newcomers)

- when using custom entropy, pressing the "enter" key generate the wallet (click on "generate wallet" button is always available)

- aeon prefix value set to "Wm" instead of "W"

### CODE REVIEW

File: monero-wallet-generator.html [hunk 1]
```diff
@@ -10743,20 +10742,17 @@ <h4 style="color: #555;">Offline wallet generator</h4>
                 <input class="btn orange" type="button" onclick="js:genwallet(null);"
                        value="Generate wallet" action=""/>
                 <p>Custom entropy (leave empty to use PRNG)</p>
-                <input type="text" value="" id="user_entropy_widget" />
+                <input type="text" value="" id="user_entropy_widget" onKeyPress="if (event.keyCode == 13) js:genwallet(null);" />
         </div>
         <div class="col s1 offset-s1"><h4>or</h4></div>
         <div class="col s5 offset-s1">
-            <form>
```
Comment (moneromooo-monero) [inline monero-wallet-generator.html]: Why are these removed ?

File: monero-wallet-generator.html [hunk 2]
```diff
@@ -10938,14 +10933,23 @@ <h5 id="gpg-instructions">How to verify GPG signatures</h5>
 </footer>
 
 <script>
+
+if (document.getElementById("currencyDropDownList").value == "Monero") {
+  var cnUtil = cnUtilGen(moneroConfig);
```
Comment (moneromooo-monero) [inline monero-wallet-generator.html]: Does this not add a *local* variable ?

File: monero-wallet-generator.html.asc [hunk 3]
```diff
@@ -1,17 +0,0 @@
...
-dS+i2cp4GATZGqH5dxYBkKq7xlqN1/+GRx9bhvIa6fanI99ldm9ajmh7Ehd24Khl
-0lR/ngXk8HKWyRlNtVuUZ47i9lkUibygRVtL2HZ2ML+MUZrNRH3Hk5WJp/FyjQ8Y
-LFt/DVCBAf7o+Ohjrl/w5/SI4baY2B5YZhOKD5pWMlRfgZZp+aIWDDAwgYEUrZN6
-/gnfTrOLIWgsm+eZjMvVUopJrVDrinTkS5d03F8Yy9DH2P1qcF102dzHnK0bAwxR
-6BAwnCqYYsIZpKVeq/JUTC6OtQt3LImQzhbATCTAGEoSNs7zwQ600r1QkyzAJwFv
-FmJM4gTX09lISbQZKDPu0DAO967TTK8JQqNtGZ2ibFVVBoQ8posxJjnf2drjB5nD
-hMHc0vD9nE2NoTp+t/scOZw/avSGfvuSPmcBlpkWJqqZZG7o8RGFKt89qZlcIRQ8
-jt7ihU/WJxDiA5Qtwu2R
-=xteP
------END PGP SIGNATURE-----
```
Comment (moneromooo-monero) [inline monero-wallet-generator.html.asc]: Please do not remove this

File: monero-wallet-generator.html (hunk 2 above)
Comment (jonathancross) [reply monero-wallet-generator.html]: It is global in JS (if it were in a function it would be local).
Code is not so readable though -- better to declare above then assign value inside the if / else (without the `var` keyword of course).

File: monero-wallet-generator.html [hunk 4]
```diff
@@ -10938,14 +10933,23 @@ <h5 id="gpg-instructions">How to verify GPG signatures</h5>
...
+  current_lang='english';
+}
+else if (document.getElementById("currencyDropDownList").value == "Aeon") {
+  var cnUtil = cnUtilGen(aeonConfig);
+  current_lang='electrum';
```
Comment (jonathancross) [inline monero-wallet-generator.html]: Nit: Please add spaces on either side of the `=`

File: monero-wallet-generator.html [hunk 5]
```diff
@@ -10938,14 +10933,23 @@ <h5 id="gpg-instructions">How to verify GPG signatures</h5>
...
+};
```
Comment (jonathancross) [inline monero-wallet-generator.html]: Nit: Please remove this unnecessary semicolon.

File: monero-wallet-generator.html (hunk 1 above)
Comment (jonathancross) [reply monero-wallet-generator.html]: I'm concerned about this too as this html file has no doctype and therefore may not be interpreted as HTML5 https://stackoverflow.com/a/22328498/1117929

Form elements in older html specs need to have a <form> element parent.

File: monero-wallet-generator.html (hunk 5 above)
Comment (Arux-BTT) [reply monero-wallet-generator.html]: Done.

File: monero-wallet-generator.html (hunk 4 above)
Comment (Arux-BTT) [reply monero-wallet-generator.html]: Done.

File: monero-wallet-generator.html.asc (hunk 3 above)
Comment (Arux-BTT) [reply monero-wallet-generator.html.asc]: My bad! I was worried to keep your signature on my repo. i was thinking that it's confusing if my commit is with your signature.

File: monero-wallet-generator.html (hunk 1 above)
Comment (Arux-BTT) [reply monero-wallet-generator.html]: It was removed because if you let it, when you press enter key (instead of click on "generate wallet with prefix" button), the page is reloaded and the wallet is not created.
The page don't need to connect to a server so I don't think form is needed (https://stackoverflow.com/questions/8568660/is-there-any-danger-to-using-input-fields-outside-without-forms-in-html-javascri) but I could be wrong, I'm absolutely not a javascript expert. I only read the manual.
About the doctype, i'm too inexperienced to answer efficiently.

Comment (moneromooo-monero) [reply monero-wallet-generator.html]: That all looks OK to me. If there's a problem with a doctype, it can be added later.

**Comment by moneromooo-monero** [comment]
Can you rebase to latest please ? There are changes from saddam conflicting with your changes.

**Comment by Arux-BTT** [comment]
I'm sorry, I didn't find any elegant solution to rebase properly. The code is ok but history is not.
I don't want to pollute your commit history with 5 commits in the mess. I think it's better (if the code satisfy you) to close this PR and I will rewrite an another one with same code but clean history on the top of latest master.
